salt/config/__init__.py

```diff
@@ -1100,7 +1090,7 @@ def _gather_buffer_space():
     'minion_id_caching': True,
     'keysize': 2048,
     'transport': 'zeromq',
-    'auth_timeout': 5,
+    'auth_timeout': 60,
```

Comment:
We just changed this as well. Why is this being changed back? Here's the discussion: https://github.com/saltstack/salt/pull/35463